SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs

```diff
@@ -73,7 +73,7 @@ public class SpeechSandboxStreaming : MonoBehaviour
     private string assistantUsername;
     [Tooltip("The authentication password.")]
     [SerializeField]
-    private string assistantPassword;
+    private string assitantPassword;
```

Comment:
I'm not sure what you are trying to do here? You are just changing the name, and it is now misspelled.

Reply:
Actually, the IBM cloud has removed the username and password system. So, I have just made changes to your code with the latest documentation on the IBM cloud which uses only API key and IAM URL .
I will correct the spelling.